charts/fluent-bit/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
   - logging
   - fluent-bit
   - fluentd
-version: 0.19.5
+version: 0.20.0
```

Comment:
As a non-breaking change does this need to be `0.20.0` or should it be `0.19.6`?

Reply:
@naseemkullah?

Reply:
Both breaking changes and new features can be minor bumps until we bite the gun and make a 1.x.x release imho

Reply:
Technically in SemVer all version changes before `v1.0.0` are breaking, but I tend prefer to follow the `0.major.minor` pattern and as such I'd prefer `0.19.6`. Happy either way as both are technically correct.

Reply:
ok - no preference on my end, merge at will @stevehipwell! 🚀 

Thanks @elialgranti